docs/components/query-frontend.md

```diff
@@ -89,7 +136,7 @@ Flags:
                               priority). Content of YAML file with tracing
                               configuration. See format details:
                               https://thanos.io/tip/tracing.md/#configuration
-      --query-range.split-interval=24h
+      --query-range.split-queries-by-interval=24h
```

Comment:
`split-interval` is fine IMHO, but not sure.

If we want to split other endpoints queries in the future (labels, series, etc), do we need separate split intervals for different endpoints?

Reply:
again I want this to be the same the flags upstream